physics/noahmpdrv.F90

```diff
@@ -37,6 +37,7 @@ subroutine noahmpdrv_init(lsm, lsm_noahmp, me, isot, ivegsrc, &
         use machine,          only: kind_phys
         use set_soilveg_mod,  only: set_soilveg
         use namelist_soilveg
+        use noahmp_tables
```

Comment:
Please list what is being imported using "only"

Reply:
For this case we need import all functions from module noahmp_tables